nifi-bootstrap/src/main/java/org/apache/nifi/bootstrap/RunNiFi.java

```diff
@@ -221,29 +224,45 @@ public static void main(String[] args) throws IOException, InterruptedException
                 verbose = false;
             }
         } else if (cmd.equalsIgnoreCase("status-history")) {
-            if (args.length != 4) {
-                System.err.printf("Wrong number of arguments: %d instead of 4, the command parameters are: " +
-                        "status-history --days <number of days> <dumpFile>%n", args.length);
-                System.exit(0);
+            if (args.length < 2) {
+                System.err.printf("Wrong number of arguments: %d instead of 3 or 4, the command parameters are: " +
```

Comment:
I'll change this later to 1 or 2. I'll need to rebase to main later anyway to fix the contrib check.